src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/steam/GregtechMetaTileEntity_SteamMacerator.java

```diff
@@ -123,4 +129,24 @@ public GT_Recipe.GT_Recipe_Map getRecipeMap() {
         return GT_Recipe.GT_Recipe_Map.sMaceratorRecipes;
     }
 
+    @Override
+    protected ProcessingLogic createProcessingLogic() {
+        return new ProcessingLogic() {
+
+            @Override
+            @Nonnull
+            public GT_ParallelHelper createParallelHelper(@Nonnull GT_Recipe recipe) {
+                return super.createParallelHelper(recipe).setCustomItemOutputCalculation(parallel -> {
+                    ArrayList<ItemStack> items = new ArrayList<>();
+                    ItemStack output = recipe.getOutput(0);
+                    if (output != null) {
+                        output = output.copy();
+                        output.stackSize *= parallel;
```

Comment:
Can this handle e.g. macerating iron block? (It would output 8*9=72 dust)

Reply:
yes. I iron block only has 1 output. This won't handle machines being macerated tho.

Reply:
What I mean is that the stacksize of output would reach 72 dusts. Will the machine be able to actually output 72 dusts as 64 + 8 or will it cut off the 8 or another issue?

Reply:
It will be able to output it.

Reply:
After all. the originalStacksize*parallel was used before to lower chance rolling, but was now changed to not do that.